Makefile

```diff
@@ -35,8 +35,7 @@ install: manifests
 	kubectl apply -f config/crd/bases
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: manifests
-	kubectl apply -f config/crd/bases
```

Comment:
Would it hurt to redeploy the CRD? What if you've never deployed the CRD in your test cluster?

Reply:
@tekenstam, it looks like the `deploy` target will call `install` target as a dependency (after the `:` on line 38), so I believe a deploy will include an install step. In fact, given the dependency chain, it looks like the order of targets applied leading up to a `deploy` would be:

`controller-gen`->`manifests`->`install`->`deploy`